website/routes.py

```diff
@@ -789,6 +805,22 @@ def make_url_map(app):
             OsfWebRenderer('public/forgot_password.mako', trust=False)
         ),
 
+        # institutional reset password get
+        Rule(
+            '/forgotpassword-institution/',
+            'get',
+            auth_views.forgot_password_institution_get,
+            OsfWebRenderer('public/forgot_password_institution.mako', trust=False)
+        ),
```

Comment:
As mentioned earlier, remote this routes (thus 404 if hit) or redirect to CAS

Reply:
:+1: